rpp30-boc-meltdown/biteshare
# Guest list

## Description

Stub out the scrollable guest list component UI.

## How to test

Pull this branch, run the bundle, and it should at first look like this:
![Screen Shot 2021-12-22 at 1 24 36 PM](https://user-images.githubusercontent.com/43324065/147154989-855a48bd-de58-48cb-8f90-bd313976fb9e.png)

Clicking on 'Allow' should hide the two buttons for that guest and display a status indicator default to 'Not Ready':
![Screen Shot 2021-12-22 at 1 24 55 PM](https://user-images.githubusercontent.com/43324065/147155093-4fcbaeb1-8d10-41e3-a8b3-e96a88dde88b.png)

Clicking on 'Deny' should remove the guest from the list (I denied Ava so she is not in the list anymore):
![Screen Shot 2021-12-22 at 1 25 11 PM](https://user-images.githubusercontent.com/43324065/147155143-abfff842-ec9b-43ae-8ad4-b9c1466e1aa0.png)


## Notes


## Issue tracking

https://trello.com/c/rkb1nSM6
https://trello.com/c/XFbeVqpu
